api/internal/utils/consts/error.go

```diff
@@ -16,13 +16,33 @@
  */
 package consts
 
-import "github.com/shiningrush/droplet/data"
+import (
+	"errors"
+
+	"github.com/shiningrush/droplet/data"
+)
 
 const (
 	ErrBadRequest = 20001
 	ErrForbidden  = 20002
 )
 
+const (
+	UpstreamIDNotFound = "upstream id: %s not found"
+	ServiceIDNotFound  = "service id: %s not found"
+)
+
+var (
+	ErrorUsernamePassword         = errors.New("username or password error")
```

Comment:
It's a good coding style to give each exportable variable a comment paragraph to explain its meaning.

Reply:
Good idea.

Reply:
Updated.